osbs/api.py

```diff
@@ -553,6 +558,29 @@ def create_worker_build(self, *args, **kwargs):
         return self.create_prod_build(*args, **kwargs)
 
     @osbsapi
+    def create_orchestrator_build(self, *args, **kwargs):
+        """
+        Create an orchestrator build
+
+        Pass through method to create_prod_build with the following
+        modifications:
+            - platforms param is required
+            - inner template set to orchestrator_inner.json if not set
+            - outer template set to orchestrator.json if not set
+            - customize configuration set to orchestrator_customize.json if not set
+
+        :return: BuildResponse instance
+        """
+        if not kwargs.get('platforms'):
```

Comment:
I see you explicitly listed it as a failing value in the tests.

Given that for multi-cluster there would be no Koji build created, and for multi-arch it would try to create an empty build, it's best to catch this error early like this.